webui.py

```
@@ -39,6 +39,26 @@
 import enhanced.comfy_task as comfy_task
 from enhanced.simpleai import comfyd
 
+import json
```

Ruff (0.8.2)

Redefinition of unused `json` from line 6

Remove definition: `json`

(F811)